README.rst

```diff
@@ -73,61 +111,72 @@ Prerequisites
+   | |jpegturbo link|_ or later             | *This can be unofficially disabled. See below.*                                             |
+   +----------------------------------------+---------------------------------------------------------------------------------------------+
+   | |ffmpeg link|_ or later                | We recommend using version 3.4.2 compiled following the *instructions below*.               |
    +----------------------------------------+---------------------------------------------------------------------------------------------+
-   | |opencv link|_ or later                | | We recommend using version 3.4+, however previous versions are also compatible.           |
-   |                                        | | *OpenCV 2.x compatibility is provided unofficially*                                       |
+   | |opencv link|_ or later                | | We recommend using version 3.4+. However previous versions are also compatible.           |
+   |                                        | | *OpenCV 2.x compatibility is provided unofficially.*                                      |
    +----------------------------------------+---------------------------------------------------------------------------------------------+
-   | **(Optional)** |lmdb link|_ or later   |                                                                                             |
+   | (Optional) |lmdb link|_ or later       |                                                                                             |
    +----------------------------------------+---------------------------------------------------------------------------------------------+
    | One or more of the following Deep Learning frameworks:                                                                               |
-   |      -  |mxnet link|_ ``mxnet-cu90`` or later                                                                                        |
-   |      -  |pytorch link|_                                                                                                              |
-   |      -  |tf link|_ or later                                                                                                          |
+   |      * |mxnet link|_ ``mxnet-cu90`` or later                                                                                         |
+   |      * |pytorch link|_                                                                                                               |
+   |      * |tf link|_ or later                                                                                                           |
    +----------------------------------------+---------------------------------------------------------------------------------------------+
 
-.. note::
 
-   TensorFlow installation is required to build the TensorFlow plugin for DALI
+**NOTE:**
```

Comment:
Do we want to replace note with just formatted string?

Reply:
I went back to the note admonition. (I was considering the formatted string because GitHub doesn't appear to support the admonition. )